### PR TITLE
Aave-v3 rewards

### PR DESCRIPTION
**Pull request by iri031**
Modifications in aave-v3 resolver: 
- [x] Updated and tested user's unclaimed rewards for avalanche, arbitrum, polygon, fantom, optimism.
- [x] Updated and tested isolation borrow enabled for avalanche, arbitrum, polygon, fantom, optimism.

### CODE REVIEW

File: contracts/protocols/arbitrum/aave_v3/helpers.sol [hunk 1]
```diff
@@ -190,55 +199,62 @@ contract AaveV3Helper is DSMath {
...
-                _aggregateIncentive[i].sIncentiveData.rewardsTokenInformation,
-                _aggregateUserIncentive[i].sTokenIncentivesUserData.userRewardsInformation
+                _aggregateIncentive[i].sIncentiveData.rewardsTokenInformation
             );
             RewardsInfo[] memory _vRewards = getRewardInfo(
-                _aggregateIncentive[i].vIncentiveData.rewardsTokenInformation,
-                _aggregateUserIncentive[i].vTokenIncentivesUserData.userRewardsInformation
+                _aggregateIncentive[i].vIncentiveData.rewardsTokenInformation
             );
+            rToken[0] = _aggregateIncentive[i].aIncentiveData.tokenAddress;
```
Comment (pradyuman-verma) [inline contracts/protocols/arbitrum/aave_v3/helpers.sol:229]: Why don't you pass this directly?

Comment (iri031) [reply contracts/protocols/arbitrum/aave_v3/helpers.sol:229]: The function from the RewardsController, `getAllUserRewards` takes array of assets as input.

Comment (pradyuman-verma) [reply contracts/protocols/arbitrum/aave_v3/helpers.sol:229]: Ohh okay

**Review by pradyuman-verma — APPROVED** [review]

**Comment by iri031** [comment]
Deployed contract addresses: 
Avalanche: [0xe6aa2d277aAfBb9e19354F6f893737C3608ff995](https://snowtrace.io/address/0xe6aa2d277aAfBb9e19354F6f893737C3608ff995#code)
Polygon: [0x44aC5eD26d8B7BBcF3Bcb9Ac486191cBb294B140](https://polygonscan.com/address/0x44aC5eD26d8B7BBcF3Bcb9Ac486191cBb294B140#code)
Arbitrum: [0x3344AE78b00d0D92454B0efae5285703A34cA321](https://arbiscan.io/address/0x3344ae78b00d0d92454b0efae5285703a34ca321#code)
Optimism: [0xe6aa2d277aAfBb9e19354F6f893737C3608ff995](https://optimistic.etherscan.io/address/0xe6aa2d277aAfBb9e19354F6f893737C3608ff995#code)
Fantom: [0xa720C0B9A659817ecc0807b74D8e7Cf597268740](https://ftmscan.com/address/0xa720C0B9A659817ecc0807b74D8e7Cf597268740#code)

**Review by pradyuman-verma — APPROVED** [review]

**Pull request merged**